### ISSUE TITLE
Hi, Do support Chinese?

### ISSUE THREAD

**Issue by doudouaili** [dexter/dexter#45]
Hello, I didn't find any information about Chinese corpus in Wikipedia。

**Comment by diegoceccarelli** [comment]
No, this version can't support Chinese because it assumes ascii encoding of
the mentions. but I might able to help, wanna have a chat?
